shimmingtoolbox/pmu.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*
+#
+# Low level read of Siemens Physiological Log files
+# Adapted from https://gist.github.com/rtrhd/6172344
+#
+
+import numpy as np
+
+
+def read_resp(fname_pmu):
+    """
+    Read a Siemens Physiological Log file. Returns a tuple
+    with the logging data as numpy integer array and times
+    in the form of milliseconds past midnight.
+
+    Args:
+        fname_pmu: srt or file object
+                   a file name to open or an already opened file-like object
+    Returns:
+        tuple:
```

Comment:
wouldn't a dict provide more flexibility? Pros:
- you can easily add/insert a field without breaking everything
- you can add a description for each field (eg: key 'description', in addition to the key 'value')
- less prone to errors (eg: selecting the wrong index)

Reply:
I'm considering moving to an o.o.p. approach for the PMU as it is its own entity with multiple attributes. Adding different methods for interpolating to the acquisition's timepoints would be neatly organized within the class.

Reply:
i guess that makes sense.